src/main/scripts/org/reaktivity/specification/nukleus/tcp/streams/server.sent.data/server/target.rpt

```diff
@@ -73,6 +70,3 @@ write flush
 read [0x40 0x00 0x00 0x02]
 read ${newTargetOutputEstId}
 read [0..4] # TODO: read int 13 option byteorder native
```

Comment:
Allows the script to be used for a case where the write is done twice (first being a partial write). In this case two window update frames are received. I felt this was a case of over specification. It's perfectly valid to get several window updates.